lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_state_manager.cpp

```diff
@@ -130,5 +148,14 @@ status_code_e SgwStateManager::read_ue_state_from_db() {
 sgw_state_t* SgwStateManager::get_state(bool read_from_db) {
   return state_cache_p;
 }
+
+map_uint32_sgw_eps_bearer_context_t* SgwStateManager::get_s8_state_teid_map() {
+  AssertFatal(
```

Comment:
Use Error log instead of assertFatal

Reply:
It's mandatory to have this flag, is_initialized to be initialized. So I think assertFatal should be fine

Reply:
Yeah, if the state is not cleanly initialized from Redis, the service should fail to start up.